x/ccv/consumer/ibc_module.go

```diff
@@ -94,10 +103,10 @@ func (am AppModule) OnChanOpenAck(
 	ctx sdk.Context,
 	portID,
 	channelID string,
-	counterpartyChannelID string,
+	_ string, // Counter party channel ID is unused per spec
```

Comment:
Curious why the IBCModule interface enforces us to pass in a counter party channel ID here? [Spec](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/methods.md#ccv-ccf-coack1) doesn't mention it. Is there uses for it in other IBC modules?

Reply:
@AdityaSripal should we check here the `counterpartyChannelID`? Or is it already done by core IBC?